website/frontend/src/components/Footer.js

```diff
@@ -94,6 +96,83 @@ const Footer = () => {
     setSelectedCountry(currentAirqloud);
   }, [currentAirqloud]);
 
+  useEffect(() => {
+    const getUserCountryFromStorage = () => {
+      const storedCountry = localStorage.getItem('userCountry');
+      if (storedCountry) {
+        dispatch(setCurrentAirQloudData(storedCountry));
+      } else {
+        getUserGeolocation();
+      }
+    };
+
+    // Function to fetch users country based on their IP address
+    const getUserCountry = async (latitude, longitude) => {
+      try {
+        const response = await axios.get(
+          `https://api.bigdatacloud.net/data/reverse-geocode-client?latitude=${latitude}&longitude=${longitude}&localityLanguage=en`
```

Comment:
thanks @OchiengPaul442 ,can we have this url stored somewhere else? and just referenced here as a variable? In our codebase, do we have somewhere we keep such constants like urls? Thanks again!